socksx/examples/client.rs

```diff
@@ -52,34 +59,35 @@ async fn main() -> Result<()> {
     }
 }
 
-///
-///
-///
+/// Connects to a destination using SOCKS5 protocol.
```

Comment:
Well, there's no need to :), but I tend to write function docstrings a bit more verbose to communicate what arguments and returns values do as well. See the suggestion for an example. But I get this may also be way too much effort xD and what you have is most certainly better than nothing

```suggestion
/// Connects to a destination through a proxy using SOCKS5 protocol, then sends an example message through the network tunnel.
/// 
/// # Arguments
/// - `proxy_addr`: The address of the SOCKS5 proxy through which the traffic will be tunnelled.
/// - `dest_addr`: The address to which the traffic should be sent after the proxy.
/// 
/// # Errors
/// This function can error if we failed to connect to the given proxy or failed to send it an example message.
```

Reply:
I see later on you did this in other places, awesome! Still, you could do something similar here for consistency if you're still willing to write them

Reply:
Thank you for the example.